.github/workflows/build-binaries.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Binaries
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+  workflow_dispatch:
```

Comment:
I am assuming this workflow would run every time a new tag is pushed or updated?

Reply:
just changed to only '*'  - but yeah, the workflow should be triggered when any new tag is pushed

Reply:
...or could be triggered manually

Reply:
I don't think we should build on all tags. Should keep the regex at `v*`?

Reply:
yeah, we can try with only that for now - changed